tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/connectors/sql/transaction/SQLTransactionsTest.java

```diff
@@ -46,65 +47,109 @@ public void setup() {
         SQLDBUtils.initHSQLDBDatabase(SQLDBUtils.DB_DIRECTORY, DB_NAME, "datafiles/sql/SQLTableCreate.sql");
     }
 
-    @Test(groups = "TransactionTest")
+    @Test(groups = TRANSACTION_TEST_GROUP)
     public void testLocalTransacton() {
         BValue[] returns = BRunUtil.invoke(result, "testLocalTransacton");
         Assert.assertEquals(((BInteger) returns[0]).intValue(), 0);
         Assert.assertEquals(((BInteger) returns[1]).intValue(), 2);
     }
 
-    @Test(groups = "TransactionTest")
+    @Test(groups = TRANSACTION_TEST_GROUP)
     public void testTransactonRollback() {
         BValue[] returns = BRunUtil.invoke(result, "testTransactonRollback");
         Assert.assertEquals(((BInteger) returns[0]).intValue(), -1);
         Assert.assertEquals(((BInteger) returns[1]).intValue(), 0);
     }
 
-    @Test(groups = "TransactionTest")
+    @Test(groups = TRANSACTION_TEST_GROUP)
+    public void testLocalTransactionUpdateWithGeneratedKeys() {
+        BValue[] returns = BRunUtil.invoke(result, "testLocalTransactionUpdateWithGeneratedKeys");
+        Assert.assertEquals(((BInteger) returns[0]).intValue(), 0);
```

Comment:
I would be better if we can add a message for each situation, so when in a fail we can easily identify.